README.md

```diff
@@ -126,34 +131,65 @@ func (s *server) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
     // Chose a supplier endpoint
     selectedSupplier := sessionSupplierEndpoints.SuppliersEndpoints[0]
 
-    // Read the request body
-    requestBodyBz, err := io.ReadAll(request.Body)
-    if err != nil {
-        // Handle error
-    }
-    request.Body.Close()
+    // Serialize the whole upstream request
```

Comment:
I'm a bigger fan of example test files: https://github.com/pokt-network/smt/blob/main/smt_example_test.go

It scales better as things change. 

Can we update it?